kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/MMModem3gppRegistrationState.java

```diff
@@ -0,0 +1,108 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.nm;
+
+import org.eclipse.kura.net.status.modem.RegistrationStatus;
+import org.freedesktop.dbus.types.UInt32;
+
+public enum MMModem3gppRegistrationState {
+
+    MM_MODEM_3GPP_REGISTRATION_STATE_IDLE(0),
+    MM_MODEM_3GPP_REGISTRATION_STATE_HOME(1),
+    MM_MODEM_3GPP_REGISTRATION_STATE_SEARCHING(2),
+    MM_MODEM_3GPP_REGISTRATION_STATE_DENIED(3),
+    MM_MODEM_3GPP_REGISTRATION_STATE_UNKNOWN(4),
+    MM_MODEM_3GPP_REGISTRATION_STATE_ROAMING(5),
+    MM_MODEM_3GPP_REGISTRATION_STATE_HOME_SMS_ONLY(6),
+    MM_MODEM_3GPP_REGISTRATION_STATE_ROAMING_SMS_ONLY(7),
+    MM_MODEM_3GPP_REGISTRATION_STATE_EMERGENCY_ONLY(8),
+    MM_MODEM_3GPP_REGISTRATION_STATE_HOME_CSFB_NOT_PREFERRED(9),
+    MM_MODEM_3GPP_REGISTRATION_STATE_ROAMING_CSFB_NOT_PREFERRED(10),
+    MM_MODEM_3GPP_REGISTRATION_STATE_ATTACHED_RLOS(11);
```

Comment:
Why did you add a value for all enum if you don't use it? Is it a good idea to have values for the enums?